multiversx/mx-chain-go
# keep-SCR-creation-order

## Reasoning behind the pull request
SCRs are not kept in order for execution.
  
## Proposed changes
Sorted SCRs by the index they were created instead of hashes

## Testing procedure
Send multiple NFTs cross shard and check whether the order of execution on destination shard is the same as in sender shard.

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
